src/job.rs

```diff
@@ -0,0 +1,62 @@
+use crate::glue;
+use roc_std::{RocList, RocStr};
+use std::hash::{Hash, Hasher};
+use std::process::Command;
+
+#[derive(Debug, Eq, Hash, PartialEq, Clone, Copy)]
+pub struct Id(u64);
+
+impl From<u64> for Id {
```

Comment:
nit: Instead of from, I would suggest making some sort of "new" method. I feel that in general that is more clear that these are distinct.

Or even just nothing and make it explicit. Id(some_u64). That feels clearer than enabling into for this.

Reply:
well, turns out we weren't using this anymore anyway 😆

Reply:
removed in eba18a5, the new commit from that force-push